packages/scene-composer/src/components/StateManager.tsx

```diff
@@ -234,7 +234,7 @@ const StateManager: React.FC<SceneComposerInternalProps> = ({
   }, [sceneContentUri]);
 
   useEffect(() => {
-    if (enableMatterportViewer && sceneMetadataModule && matterportModelId) {
+    if (sceneMetadataModule && matterportModelId) {
```

Comment:
will set default value of enableMatterportViewer be `true`  solve the issue too?